prow/config/config.go

```diff
@@ -613,10 +613,14 @@ type Deck struct {
 	Branding *Branding `json:"branding,omitempty"`
 	// GoogleAnalytics, if specified, include a Google Analytics tracking code on each page.
 	GoogleAnalytics string `json:"google_analytics,omitempty"`
-	// RerunAuthConfig specifies who is able to trigger job reruns if that feature is enabled.
-	// The permissions here apply to all jobs. GitHub teams are not yet supported
-	// for the global Deck config.
-	RerunAuthConfig prowapi.RerunAuthConfig `json:"rerun_auth_config,omitempty"`
+	// Deprecated: RerunAuthConfig specifies who is able to trigger job reruns if that feature is enabled.
+	// The permissions here apply to all jobs.
+	// This option will be removed in favor of RerunAuthConfigs in March 2020.
```

Comment:
March 2020 is probably not realistic anymore?

Reply:
Oops, this was actually supposed to be May 2020, like the other deprecations. Update this reference and the others  to `July 2020`.